toolchain/sem_ir/inst_kind.h

```diff
@@ -73,13 +73,17 @@ enum class InstConstantKind : int8_t {
   // same as `WheneverPossible`, except that the operands are known in advance
   // to always be constant. For example, `IntValue`.
   Always,
+  // The instruction may be a unique constant, as described below for `Unique`.
+  // Otherwise the instruction is not constant. This is used for `VarStorage`,
+  // where global variables are `Unique` and other variables are non-constant.
```

Comment:
I sabotaged you
```suggestion
  // The instruction may be a unique constant, as described below for
  // `AlwaysUnique`. Otherwise the instruction is not constant. This is used for
  // `VarStorage`, where global variables are `Unique` and other variables are
  // non-constant.
```